src/components/Lend/LoanSearch/ChallengeHeader.vue

```diff
@@ -154,6 +157,17 @@ export default {
 </script>
 
 <style scoped lang="postcss">
+
+.loan-activity-overlay {
+	background: linear-gradient(90deg, rgba(245, 245, 245,0) 0%,rgba(245, 245, 245, 1) 90%);
+}
+
+@screen lg {
+	.loan-activity-overlay {
+		background: linear-gradient(90deg, rgba(245, 245, 245,0) 0%,rgba(245, 245, 245, 1) 70%);
```

Comment:
Just out of curiosity, why is the slight opacity difference needed between desktop and mobile?

Reply:
Also verify this on tablet, we are using same condition for mobile but is larger than mobile and desktop of course.
Using a fixed size instead of a percentage for all cases could be a better approach maybe...